paper/basic_training.tex

```diff
@@ -730,11 +731,170 @@ \subsubsection{Summary}
 \subsection{Barostats}
 \label{sec:barostats}
 \begin{itemize}
-\item What is it?
+\item Motivation
+\item Background
 \item Brief description of how it works
 \item Popular barostats
+\item Summary
 \end{itemize}
 
+\subsubsection{Motivation}
+When browsing the literature for many physical properties of various
+materials, you will observe that a multitude of the thermodynamic
+properties of interest are measured under some contstant temperature
+and pressure. Whether it is standard temperature and pressure (STP), or
+a more extreme value, many experimental studies are performed under
+these conditions. Termed the isothermal-isobaric ensemble, this
+might be one of the most popular ensembles for simulationists in
+general. As is the case with thermostats, if the pressure must be
+maintained in a simulation, a barostat algorithm will be needed to
+sample this ensemble. This section will review the background of the
+barostats, a general overview of how they work, and introduce some
+popular barostats.
+
+\subsubsection{Background and How They Work}
+Recall that in experimental studies, to control the pressure,
```

Comment:
I feel like this paragraph will be slightly confusing to novices, at least at first. Specifically, a lab experiment done (over a short time) in an open container is nominally "constant pressure" but the wording here says "some device must be used to control the volume of the system", which will likely read (to novices) like lab conditions are NOT constant pressure. But then your final sentence in this paragraph says that if the system can exchange volume with the surroundings it is at constant pressure, which seems opposite of the statement that "some device must be used..."